src/main/java/brokeculator/command/Command.java

```diff
@@ -1,5 +1,6 @@
 package brokeculator.command;
 
 public abstract class Command {
```

Comment:
To my knowledge abstract classes are more like objects (not in the Java sense) while interfaces are more like concepts, so our Saveable is an interface since it describes anything that is saveable while Command can be an abstract class since a command is a "thing"